docs/x509/reference.rst

```diff
@@ -2404,6 +2404,7 @@ X.509 Extensions
         .. doctest::
 
             >>> from cryptography import x509
+            >>> from cryptography.x509.oid import ExtensionOID
             >>> from cryptography.hazmat.primitives import hashes
```

Comment:
```suggestion
            >>> from cryptography.hazmat.primitives import hashes
            >>> from cryptography.x509.oid import ExtensionOID
```

Small nit to keep teh imports ordered correctly

Reply:
Applied.